style.css

```
@@ -1 +1,263 @@
+* {
+ box-sizing: border-box;
+}
+.body {
+ background-color: #e6e2e1;
+}
 
+/*Nav Bar*/
+
+.nav-flex {
+ display: flex;
+ flex-direction: row;
+ justify-content: flex-end;
+ width: 100%;
+ height: 200px;
+ vertical-align: middle;
+}
+.sign {
+ background-color: #3665d9;
+ font-size: 15px;
+ font-family: Verdana, Geneva, Tahoma, sans-serif;
+ color: white;
+ margin: 4px;
+ border-radius: 5px;
+ border: 1px solid black;
+ width: 110px;
+ height: 35px;
+}
+#block {
+ margin-right: 6px;
+}
+.nav-align p a {
+ color: #33ace8;
+ text-decoration: none;
+}
+.nav-align p {
+ padding: 0px;
+ margin: 2px;
+}
+
+/* form section */
+
+.main-col {
+ width: 100%;
+ height: auto;
+}
+.flex-container {
+ display: flex;
+ flex-direction: column;
+ align-items: center;
+ justify-content: center;
+ margin-top: -100px;
+}
+.flex-sub-logo {
+ display: flex;
+ flex-direction: row;
+ flex-flow: row;
+ width: 100%;
+ height: auto;
+ align-items: center;
+ justify-content: center;
+}
+.flex-items {
+ align-items: center;
+ justify-content: space-between;
+}
+.intuit {
+ width: 100px;
+ height: 30px;
+ margin-bottom: 15px;
+}
+.turbo {
+ width: 26px;
+ height: 26px;
+}
+.qbooks {
+ width: 30px;
+ height: 30px;
+ margin-left: 10px;
+}
+.mint {
+ width: 28px;
+ height: 30px;
+ margin-left: 10px;
+}
+.tag-sub {
+ color: #5c5958;
+ font-family: Verdana, Geneva, Tahoma, sans-serif;
+ font-size: 12px;
+ font-weight: lighter;
+ text-align: left;
+ margin: -20px 5px 20px 3px;
+}
+.form {
+ background-color: white;
+ width: 370px;
+ height: auto;
+ margin-top: 15px;
+ border: 1.3px solid #c3c5c7;
+ border-radius: 3px;
+}
+.h4-header {
+ color: #5c5958;
+ font-family: Verdana, Geneva, Tahoma, sans-serif;
+ font-size: 25px;
+ font-weight: lighter;
+ text-align: center;
+ padding-top: 5px;
+}
+.tag-up {
+ color: #858382;
+ font-family: Verdana, Geneva, Tahoma, sans-serif;
+ font-size: 14px;
+ font-weight: lighter;
+ text-align: center;
+ padding-left: 25px;
+ padding-right: 25px;
+ margin-top: -15px;
+}
+.sub-font {
+ font-style: italic;
+ text-align: center;
+ font-weight: lighter;
+ color: #858382;
+}
+.form a {
+ align-items: center;
+ font-size: 12px;
+ text-decoration: none;
+}
+a:hover {
+ color: #0678bf;
+ text-decoration: none;
+}
+a:link {
+ color: #34b2fa;
+ text-decoration: none;
+}
+a:visited {
+ color: #34b2fa;
+ text-decoration: none;
+}
+.account-info {
+ align-items: center;
+ border: hidden;
+}
+.account-info label {
+ cursor: pointer;
+ color: #575551;
+ padding-top: 10px;
+ padding-bottom: 10px;
+}
+.account-info input {
+ border: 1px solid #c6c9cc;
+ border-radius: 5px;
+ color: #888;
+ display: block;
+ margin: 5px 0 27px 0;
+ padding: 5px 8px;
+ width: 100%;
+}
+
+.account-info input[type="email"] {
+ padding-left: 10px;
+ padding-bottom: 8px;
+ margin-bottom: 6px;
+}
+.account-info input[type="tel"] {
+ padding-bottom: 8px;
+ padding-left: 10px;
+}
+.account-info input[type="password"] {
+ padding-bottom: 8px;
+ padding-left: 10px;
+}
```

You are repeating yourself.
```suggestion
.account-info input[type="email"],
.account-info input[type="tel"]
.account-info input[type="password"] {
  padding-bottom: 8px;
  padding-left: 10px;
}

.account-info input[type="email"] { margin-bottom: 6px; }
```